roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -2,7 +2,7 @@
 username = "admin"
 password = "changeme"
 external_capsules = []
-external_capsule_ids = `hammer -u #{username} -p #{password} --csv capsule list --search 'feature = \"Pulp Node\"' | tail -n+2 | awk -F, {'print $1'}`
+external_capsule_ids = `hammer -u #{username} -p #{password} --csv capsule list --search 'feature = \"Pulp Node\"' | grep -v "Warning:" | tail -n+2 | awk -F, {'print $1'}`
```

Comment:
Can you explain why this change was made? this doesn't seem to be related to this PR.

Reply:
So on 6.3 hammer is displaying a warning for the first line of the hammer command output. The `tail` command here is expecting everything after the second line, which makes sense for how the normal hammer csv output is, but the warning is bumping everything down a line. I added this to remove any lines containing `Warning:` so the count remains correct.

Reply:
Which warning, is there a BZ about that? :-)